lab-james/app/entry.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+// require('./scss/main.scss')
+
+require('angular')
+.module('ngAdventure', []);
+
+require('./service/map-service.js');
```

Comment:
We've been using the require.context() syntax for the previous labs, I suggest doing it this way so that you wouldn't need to worry about adding every new component.